src/Microsoft.OData.Core/ODataDeserializer.cs

```diff
@@ -68,6 +68,17 @@ internal IEdmModel Model
             }
         }
 
+        /// <summary>
+        /// Get the absolute uri
+        /// </summary>
+        internal string AbsoluteUri
+        {
+            get
+            {
+                return this.inputContext?.MessageReaderSettings?.BaseUri?.AbsoluteUri;

```

Comment:
Given the other properties, I think it is safe to assume `inputContext` is not `null`

Reply:
inputContext is from te ctor and there's a null check but using Debug.Assert.